rate-limit/index.js

```diff
@@ -0,0 +1,39 @@
+var catbox = require("catbox");
+var catboxMemory = require("catbox-memory");
+var catboxRedis = require("catbox-redis");
+
+module.exports = function(config, ttl, ready) {
+
+  var catboxEngine = config ? new catboxRedis(config) : new catboxMemory();
+
+  var client = new catbox.Client(catboxEngine);
+
+  client.start(function(err) {
```

Comment:
I think this file should do away; have processor.js call `start()` on the client, and we shouldn't expose our own interface for caching, lets just have the send_sms worker call catbox functions itself, rather than wrapping it